src/terminals.c

```diff
@@ -121,7 +121,9 @@ int gnatcoll_terminal_width(int forStderr) {
       GetStdHandle (forStderr ? STD_ERROR_HANDLE : STD_OUTPUT_HANDLE);
    CONSOLE_SCREEN_BUFFER_INFO csbiInfo;
    if (GetConsoleScreenBufferInfo (handle, &csbiInfo)) {
-      return (int)csbiInfo.dwSize.X;
+      //return (int)csbiInfo.dwSize.X; // buffer width 
+      return (int)(csbiInfo.srWindow.Right
+						-csbiInfo.srWindow.Left + 1); // window width
```

Comment:
The indentation of this line is off - please remove the tabs (same comment for line 150)